src/engraving/libmscore/lyrics.cpp

```diff
@@ -766,4 +792,16 @@ KerningType Lyrics::doComputeKerningType(const EngravingItem* nextItem) const
     }
     return KerningType::KERNING;
 }
+
+void Lyrics::removeInvalidSegments()
```

Comment:
What is considered an invalid segment? I don't really understand the purpose of this method

Reply:
That said, I have added comments to the method!